config.d

```diff
@@ -1,6 +1,7 @@
 /+
 dub.sdl:
     name "config"
+    targetPath "generated/config"
```

Comment:
Perhaps `generated/dub` is better. This might create a directory named `config`.

Reply:
Yes, exactly. You don’t want them in the same directory?